operations/tempo-mixin/alerts.libsonnet

```diff
@@ -82,11 +82,29 @@
             },
           },
           {
+            // page immediately if > 2 flushes have failed AND flush retries have failed
+            alert: 'TempoIngesterFlushesFailing',
+            expr: |||
+              sum by (%s) (increase(tempo_ingester_failed_flushes_total{}[1h])) > %s and
+              sum by (%s) (increase(tempo_ingester_failed_flushes_total{}[5m])) > 0 and
+              sum by (%s) (increase(tempo_ingester_flush_failed_retries_total{}[5m])) > 0
```

Comment:
similar thoughts here. i'm afraid this will just confuse the situation and possibly put an operator in a position where two alerts are hitting for basically the same thing.